docs/release-notes.asciidoc

```diff
@@ -69,7 +69,7 @@ See ({pull}77327[#77327]) for details.
 [[known-issues-7.10.0]]
 ==== Known issues
 
-* If you edit a rule while that rule is running, the rule fails. The rule will sometimes succeed even if UI indicates failure ({pull}82320[#82320]).
+* If you edit a rule while that rule is running, the rule fails. The rule will sometimes succeed even if the detection engine indicates a failure ({pull}82320[#82320]).
```

Comment:
Instead of :

> The rule will sometimes succeed even if the detection engine indicates a failure ({pull}82320[#82320]).

perhaps something like:

> Subsequent successful runs will retain the previous failure message ({pull}82320[#82320]).

Just to isolate the failure to being modified during execution, and that the error message will stick around even on subsequent successful runs.